temoa/extensions/modeling_to_generate_alternatives/hull.py

```
@@ -1,17 +1,31 @@
+    cv_hull: Any
+    volume: float
+    seen_norms: np.ndarray | None
+    _valid_norms: np.ndarray | None
+    tolerance: float
+    norms_checked: int
+    norms_rejected: int
+    good_points: np.ndarray | None
+    all_points: np.ndarray | None
+    norm_index: int
+
+    def __init__(self, points: np.ndarray, **kwargs: Any) -> None:
```

Ruff (0.14.10)

Unused method argument: `kwargs`

(ARG002)